bundle_update_checker.rb

```diff
@@ -1,22 +1,40 @@
 require 'net/http'
 require 'json'
 
-uri = URI('https://kits-crashlytics-com.s3.amazonaws.com/fastlane/standalone/version.json')
-json_string = Net::HTTP.get(uri)
-json = JSON.parse(json_string)
-available_bundle_version = json["bundle_version"]
+begin
+  url = URI.parse('https://kits-crashlytics-com.s3.amazonaws.com/fastlane/standalone/version.json')
+  # Work around ruby defect, where HTTP#get_response and HTTP#post_form
+  # don't use ENV proxy settings
+  # https://bugs.ruby-lang.org/issues/12724
+  http_conn = Net::HTTP.new(url.host, url.port)
+  http_conn.use_ssl = true
+  http_conn.read_timeout = 5
+  http_conn.open_timeout = 5
+  http_conn.ssl_timeout = 5
 
-path_to_version = File.expand_path(File.join(File.dirname(__FILE__), 'VERSION'))
+  resp = http_conn.request_get(url.path)
 
-version_string = File.open(path_to_version) { |file| file.each_line.first }
-current_bundle_version = version_string.chomp.to_f
+  version_file = File.expand_path(File.join(File.dirname(__FILE__), 'VERSION'))
+  unless File.exist?(version_file)
+    raise "Version file not found `#{version_file}`"
+  end
+  json_string = resp.body
+  json = JSON.parse(json_string)
+  available_bundle_version = json['bundle_version']
+  path_to_version = version_file
```

Comment:
I don't think this is necessary. We could either assign `path_to_version` above on line 17, or we could pass `version_file` below on line 26 right?